Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt

```diff
@@ -118,6 +118,15 @@
         <p tal:condition="not:portal_state/anonymous">Red titles indicate drafting-state items. Yellow titles indicate pending-state.</p>
 
         <div tal:condition="not:context/@@iterate/baseline|nothing">
+
+          <div id="representativePointField" class="field">
+            <label>Representative Point:</label>
+            <dl tal:replace="structure context/@@representative-point"></dl>
+            <div tal:condition="not:portal_state/anonymous">
```

Comment:
I think it might be useful to indicate if the repr point has changed from the baseline. Not everything in those sections is editable, and since the repr point is tied to the locations (which are in the basline/no-baseline sections) I think it should probably live there along with the locations fields.